aries_cloudagent/indy/credx/issuer.py

```diff
@@ -468,6 +468,15 @@ async def revoke_credentials(
                         tails_file_path,
                     ),
                 )
+                for cred_rev_id in rev_crids:
+                    issuer_cr_rec = await IssuerCredRevRecord.retrieve_by_ids(
+                        txn,
+                        revoc_reg_id,
+                        str(cred_rev_id),
+                    )
+                    await issuer_cr_rec.set_state(
+                        txn, IssuerCredRevRecord.STATE_REVOKED
+                    )
```

Comment:
@andrewwhitehead not sure if these updates are in the right place?  I mirrored the status updates from the Indy-sdk issuer

Reply:
I think it should probably be down below after the second `txn.handle.replace`